_src/assets/js/searchfromlocalstorage.js

```diff
@@ -37,29 +41,41 @@ function refillFormUserInfo() {
     } else if (userInfo.palette === 3) {
       addThirdPalete();
       thirdPalete.setAttribute('checked', 'true');
-    }
-
+    } //añade las clases en funcion de la info de ls
     const skillsInObject = userInfo.skills;
     const paletteUserInfo = userInfo.palette;
-    let listContentSkillsLS = "";
-    let skillsListElement = document.querySelector('.skill__tags');
-    for (let i = 0; i < skillsInObject.length; i++) {
+    const listContentSkillsLS = skillsInObject.map(skill => {
```

Comment:
Hi @arranzizquierdo !! I understood what you meant. The problem is that .map returns and array with the same length and same data type (in this case each element of skillsInObjet is a **string**). So listContentSkillsLS will return another array of strings so thats why the commas appear. If you see the skillsInObjet array, each element is separated by **commas** because the default separator is comma (,). You can change it with **array.join(separator-you-want)**. I will leave you and example below so you can try to change it!

var fruits = ["Banana", "Orange", "Apple", "Mango"];
var energy = fruits.join(" and ");

the final fruits array will be: Banana and Orange and Apple and Mango! 

You just need to find the separator we need in the array so the commas dont appear 😉...Ask me if you dont get it! 😁